cmd/main.go

```diff
@@ -479,6 +479,12 @@ func main() {
 				Scheme: ctrlMgr.GetScheme(),
 			}).SetupWithManager(ctrlMgr), "unable to create controller BMH")
 
+			log.Info("waiting for REST api readiness before starting controllers")
+			if err := apiEnabler.WaitForEnabled(time.Hour); err != nil {
```

Comment:
The timeout is not good here, we don't know when and how long it will take to upload the images.
There is a timeout on a service level so no need another one here. just wait forever.